OWNERS_ALIASES

```diff
@@ -215,13 +215,14 @@ aliases:
     - saschagrunert # SIG Technical Lead
     - tpepper # SIG Chair
   release-engineering-reviewers:
-    - cpanato # Branch Manager
-    - feiskyer # Patch Release Team
-    - hasheddan # Branch Manager
-    - hoegaarden # Patch Release Team
-    - justaugustus # SIG Chair
-    - saschagrunert # Branch Manager
-    - tpepper # SIG Chair / Patch Release Team
+    - cpanato # Release Manager
+    - feiskyer # Release Manager
+    - hasheddan # Release Manager
+    - hoegaarden # Release Manager
+    - justaugustus # SIG Chair / Release Manager
+    - saschagrunert # Release Manager
```

Comment:
@justaugustus Should we also update Sascha's entry?

```suggestion
    - saschagrunert # SIG Technical Lead / Release Manager
```

Reply:
Good catch!

Reply:
@justaugustus Done. I'll create a similar PR for k/k in a few moments. :)